src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -1519,7 +1519,12 @@ internal static string GetWinrmPluginShellName()
             // PowerShell Core uses a versioned directory to hold the plugin
             // TODO: This should be PSVersionInfo.PSVersionName once we get
             // closer to release. Right now it doesn't support alpha versions.
-            return System.String.Concat("PowerShell.", PSVersionInfo.GitCommitId);
+            string version = PSVersionInfo.GitCommitId;
```

Comment:
It would be nice if both these code paths included comments with examples demonstrating what is expected.  Otherwise it is a bit confusing.

Reply:
Will add